airflow/configuration.py

```diff
@@ -201,6 +201,11 @@ def run_command(command):
 web_server_ssl_cert =
 web_server_ssl_key =
 
+# Pass gunicorn front-end IPs allowed to handle set secure headers.
+# Multiple IPs should be comma separated.  Set to * to disable checking.
+# Useful if you are running gunicorn behind a load balancer.
+forwarded_allow_ips =
```

Comment:
Why not comment this out by default if there is no reasonable default value? Also, please do add this config to the TEST_CONFIG section as well in the same file.